regex-ballerina/build.gradle

```diff
@@ -212,3 +222,6 @@ publishing {
 
 updateTomlVersions.dependsOn unpackJballerinaTools
 build.dependsOn ballerinaBuild
+
+ballerinaPublish.dependsOn ballerinaBuild
+publish.dependsOn ballerinaPublish
```

Comment:
`test` task should be depending on `ballerinaTest`. `ballerinaBuild` should be depending on `test` task.

Reply:
Thanks for pointing it out. Reverted the deletion of the test task.